examples/howto/data_container.myst.md

```diff
@@ -11,7 +11,7 @@ kernelspec:
 ---
 
 (data_container)=
-# Using shared variables (`Data` container adaptation)
+# Using Data Containers
 
 :::{post} Dec 16, 2021
 :tags: posterior predictive, shared data 
```

Comment:
and you should also add yourself here in the author metadata. Right now the html hack that added the rendered citation at the end of a page is down, but it will be fixed, and it is not the only nice thing of using that. You will also get a page with the notebooks you have contributed too!

This is not very accessible to not put too much emphasis on authors of pages but you are free to share in on your website, social networks... Here is mine as an example: https://www.pymc.io/projects/examples/en/latest/blog/author/oriol-abril-pla.html